### ISSUE TITLE
Removing source and resulting files after the computation

### ISSUE THREAD

**Issue by darkcurrent** [fidecastro/ComfyUI-serverless#6]
Hi again,

If we are going to use ComfyUI as an API, we do not want the image or video files we send to ComfyUI for calculation or the image or video files obtained as a result of the calculation to be stored on the server. Is it possible to delete the source and result files with this code after the task is completed? If so this would be very helpful.

**Comment by fidecastro** [comment]
Good idea indeed. I'm not sure I could do it with as simple a code as what I'm doing now (which is explicitly using ComfyUI's /upload/image endpoint to upload the images). 

We could always 'brute-force' it by writing code that somehow discovers ComfyUI's folder structure, finds where the files are located, and then deletes them. This would even make the use of the uploading endpoint obsolete. 

Having said that, I feel this is lower priority than making other workflows connectable to Comfy. I'd appreciate any pull requests to make it happen though!

**Comment by WorkSync** [comment]
> Good idea indeed. I'm not sure I could do it with as simple a code as what I'm doing now (which is explicitly using ComfyUI's /upload/image endpoint to upload the images).
> 
> We could always 'brute-force' it by writing code that somehow discovers ComfyUI's folder structure, finds where the files are located, and then deletes them. This would even make the use of the uploading endpoint obsolete.
> 
> Having said that, I feel this is lower priority than making other workflows connectable to Comfy. I'd appreciate any pull requests to make it happen though!

Hello, may I ask how to write a Python code for uploading images? What should I do if I want to receive the obj file sent by the Comfyui server
